src/ahubu/lib.clj

```diff
@@ -504,14 +504,15 @@
   (omnibar-load-url url))
 
 (defn get-rc-file-raw []
-  (let [defaults (read-string (slurp "conf/default-rc"))]
-    (try
-      (conj
-       defaults
-       (read-string (slurp (format "%s/.ahuburc" (System/getProperty "user.home")))))
-      (catch Throwable t
-        (println t)
-        defaults))))
+  (let [defaults (read-string (slurp "conf/default-rc"))
+        home-rc (format "%s/.ahuburc" (System/getProperty "user.home"))
+        xdg-rc (format "%s/ahubu/ahuburc" (System/getenv "XDG_CONFIG_HOME"))]
```

Comment:
On my WM (StumpWM) this variable is not set - the XDG spec mentions if an XDG compliant app sees it is not set, we should default to $HOME/.config - can you add that in here? (maybe abstract into a defn to get-xdg-path or something).

Reply:
Oh and thanks for the PR, it's a good idea!